internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,59 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	personaldata "github.com/Yandex-Practicum/tracker/internal/personaldata"
+	spentenergy "github.com/Yandex-Practicum/tracker/internal/spentenergy"
+	usererrors "github.com/Yandex-Practicum/tracker/internal/usererrors"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	comp := strings.Split(datastring, ",")
+	if len(comp) != 2 {
+		return usererrors.ErrIncorrectNumberOfArgs
+	}
+	steps, err := strconv.Atoi(comp[0])
+	if err != nil {
+		return err
```

Comment:
ошибки лучше всегда оборачивать
```suggestion
		fmt.Errorf("invalid steps: %w", err)
```